environment.yml

```diff
@@ -2,14 +2,14 @@ name: vector
 channels:
   - conda-forge
 dependencies:
-  - jupyterlab >=1.2
```

Comment:
I believe this spacing is the Conda recommended spacing. It's used in all the Conda-forge recipe's, anyway.